README.md

```diff
@@ -347,11 +360,9 @@ declare module '@vue/composition-api' {
 </summary>
 
 ```js
-watch(() => {
-    /* ... */
-}, {
+watch(() => {}, {
   immediate: true,
-  onTrack() {}, // not available
+  onTrack() {},
```

Comment:
wow, 👍

Reply:
actually this is a mistake, probably prettier caused it :/